.circleci/config.yml

```diff
@@ -268,10 +274,18 @@ jobs:
             make test-tests
 
   test-others:
-    executor: test-executor
+    executor: test-others-executor
     steps:
       - install-ssh-alpine
       - checkout
+      - run:
+          name: "Wait until localstack container running"
+          command: |
+            timeout -t 60 sh -c \
+            'until curl --silent --fail "http://localhost:4572"; do
```

Comment:
I can't find `localstack start` command. Do we need to start localstack manually or we can just use it after this PR is merged?

Reply:
`test-others` use klaytn/build_base with localstack container. so you can see it(klaytn/build_base image) call localhost:4572 is open which is used for localstack container for S3